quickcomm/forms.py

```diff
@@ -68,13 +65,12 @@ class CreateImageForm(forms.Form):
     """A form for creating an image post."""
 
     title = forms.CharField(max_length=100)
-    source = forms.URLField(validators=[URLValidator])
-    origin = forms.URLField(validators=[URLValidator])
     description = forms.CharField(max_length=1000)
     content = forms.ImageField(validators=[validate_image_upload_format])
     categories = forms.CharField(max_length=1000)
     visibility = forms.ChoiceField(choices=Post.PostVisibility.choices)
     unlisted = forms.BooleanField(required=False)
+    friends_only=forms.BooleanField(required=False)
```

Comment:
why is this line here and in markdown post but not for text post?

as far as I can tell it's extraneous and no longer does anything